packages/backend-core/src/middleware/passport/utils.js

```diff
@@ -1,14 +1,39 @@
+const { getGlobalDB, isMultiTenant, getTenantId } = require("../../tenancy")
+const { getScopedConfig } = require("../../db/utils")
+const { Configs } = require("../../constants")
+
 /**
  * Utility to handle authentication errors.
  *
  * @param {*} done The passport callback.
  * @param {*} message Message that will be returned in the response body
  * @param {*} err (Optional) error that will be logged
  */
+
 exports.authError = function (done, message, err = null) {
   return done(
     err,
     null, // never return a user
     { message: message }
   )
 }
+
+exports.ssoCallbackUrl = async (db, config, type) => {
```

Comment:
Looks like this is a copy paste from the existing function in worker, we should remove the worker code and re-use the backend-core function to remove the duplication / potential for future bugs

Reply:
Yep, I shifted this out and should have deleted the original. Update incoming.